public/src/pages/auth/Login/Login.tsx

```diff
@@ -124,6 +124,16 @@ export const Login: FC = () => {
     getOidcClient().then((client) => setOidcClient(client));
   };
 
+  const extractLogoBgColor = (): string | undefined => {
+    const urlParams = window?.location?.href?.split('?')[1]?.split('&');
+    if (!urlParams) return undefined;
+    for (const param of urlParams) {
+      const [paramName, paramValue] = param.split('=');
+      if (paramName === 'logobgcolor') return decodeURI(paramValue);
+    }
+    return undefined;
```

Comment:
Just use the `URL` class to parse the query parameters

```suggestion
    const { searchParams } = new URL(window.location.href);
    
    return searchParams.get('logobgcolor');
```